great_expectations/profile/basic_dataset_profiler.py

```diff
@@ -150,11 +162,4 @@ def _profile(cls, dataset):
                 # print("??????", column, type_, cardinality)
                 pass
 
-        # FIXME: this is temporary hack. This expectation is failing since we are passing an empty set as an arg.
```

Comment:
expect_column_distinct_values_to_be_in_set supports None as universal set, so we don't need this hack anymore.